src/RockPaperScissors/RpsServer/Controllers/RpsController.cs

```diff
@@ -0,0 +1,210 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using RpsServer.Models;
+
+namespace RpsServer.Controllers
+{
+    [Route("api/[controller]")]
+    public class RpsController : Controller
+    {
+        private readonly GameContext context;
+
+        public RpsController(GameContext context)
+        {
+            this.context = context;
+        }
+
+        #region APIs
+        [HttpGet]
```

Comment:
Probably. My future plan for this is to correlate a user across a session. Not sure how we'll do that (login? IP?). But honestly, I'm not even using the User DB. Literally, this is just useful for creating a Guid, which I use for IDs.